conf/initializers/database.go

```diff
@@ -24,16 +31,14 @@ func SetUpDatabase() {
 	err = orm.RegisterDataBase("default", "postgres", dbURL)
 	if err != nil {
 		logs.Critical(fmt.Sprintf("Database Registration failed: %v", err))
+	} else {
+		if runMode != "test" {
+			SetUpSession(dbURL)
```

Comment:
Should we move this to another file (`initializer.go`)?
I think it shouldn't be in this file

Reply:
@hoangmirs I've moved this to `initializer.go` and set it up when successful with database setup ✨